src/app.ts

```diff
@@ -369,17 +365,20 @@ function ensureCorrectContext<B, R>(
 fedify
     // actorDispatcher uses RequestContext so doesn't need the ensureCorrectContext wrapper
     .setActorDispatcher(
-        '/.ghost/activitypub/users/{handle}',
+        '/.ghost/activitypub/users/{identifier}',
         spanWrapper(actorDispatcher(siteService, accountService)),
     )
+    .mapHandle(async () => {
+        return 'index'; // All identifiers are 'index'
```

Comment:
In the future this will be something like:
```
if (isDefaultAccount) {
  return 'index';
} else {
  return uuid;
}
```

Reply:
nice